### ISSUE TITLE
Failure validating angular.json spec with addition for i18n while running ng e2e

### ISSUE THREAD

**Issue by mtfelian** [briebug/cypress-schematic#39]
Hello. 

Having the problem on project with i18n. 
```
ng e2e
An unhandled exception occurred: Schema validation failed with the following errors:
  Data path ".projects['log-server-js']" should NOT have additional properties(i18n).
See "C:\Users\Felian\AppData\Local\Temp\ng-vHDlEJ\angular-errors.log" for further details.
```
where `log-server-js` is a project name and `angular.json` have addition for i18n:

```
"projects": {
    "log-server-js": {
      "projectType": "application",
      "schematics": {},
      "root": "",
      "sourceRoot": "src",
      "prefix": "app",
      "i18n": {
        "sourceLocale": "en-US",
        "locales": {
          "ru": "src/locale/messages.ru.xlf"
        }
      },
...
```
How to make it work using `ng e2e`?

Addition: It runs OK with following `npm run e2e` command:
```
    "cypress:run": "cypress run",
    "build:prod": "ng build --prod",
    "start:prod": "angular-http-server --path ./dist/log-server-js -a localhost -p 4200",
    "build-and-start:prod": "run-s build:prod start:prod",
    "e2e": "start-server-and-test build-and-start:prod http://localhost:4200 cypress:run"
```

**Comment by sittingbool** [comment]
same here, should be fixed to allow localize flag like in @angular-devkit/build-angular:browser

```
"build": {
          "builder": "@angular-devkit/build-angular:browser",
          "options": {
            ...
            "index": "src/index.html",
            "main": "src/main.ts",
            ....
            "localize": true // <-- this here, but probably false since e2e should have a single language
          },
```

**Comment by anthonymjones** [comment]
Closing this issue as it may be fixed as of v4.2.2. Please re-open if the issue persists.
